_posts/2016-06-30-servo-nightlies.md

```diff
@@ -0,0 +1,35 @@
+---
+layout:     post
+title:      Servo Nightly Builds Available
+date:       2016-06-30 20:00:00
+summary:    Try out the new Servo nightly builds
+categories:
+---
+
+Today the Servo team is making available nightly builds of the Servo web
+rendering engine. To make the Servo engine easy to interact with, we are
+bundling an HTML-based browser UI. While our engine is not yet web compatible,
+we want to give a larger audience the chance to start experimenting with and
+contributing to Servo. We have created packages for macOS and Linux; Windows
+and Android packages should be available soon.
+
+<iframe width="420" height="315" src="https://www.youtube.com/embed/QbHYzNVs69o" frameborder="0" allowfullscreen></iframe>
+
+## Getting started
+
+Binary packages and installation instructions are available at
+[download.servo.org](http://download.servo.org/index.html).
```

Comment:
These are redirects to Amazon and don't seem to be https capable. Maybe we can fix this later.